### PR TITLE
chore: update husky hooks

### PR DESCRIPTION
**Pull request by ersachin3112**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case
update husky hooks

### Breaking Changes
No

### Additional Info
No

### CODE REVIEW

**Review by evenstensberg — CHANGES_REQUESTED** [review]
Only update husky, dont change the commit hooks

**Comment by ersachin3112** [comment]
@evenstensberg npx is not required anymore. https://github.com/typicode/husky/releases/tag/v9.1.1

**Comment by evenstensberg** [comment]
@ersachin3112 I think its good to leave the npx commands, but okay.. For the other prs, only update husky, dont change other files except package.json and `.husky/*`

**Review by snitin315 — APPROVED** [review]
LGTM. I'm fine without `npx`, they also claim it's slightly faster.

> You can now run package commands directly, no need for npx or equivalents.
It makes writing hooks more intuitive and is also slightly faster 🐺⚡️

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "test:coverage": "npm run test:only -- --collectCoverageFrom=\"src/**/*.js\" --coverage",
     "pretest": "npm run lint",
     "test": "npm run test:coverage",
-    "prepare": "husky install && npm run build",
+    "prepare": "husky && npm run build",
```
Comment (evenstensberg) [inline package.json:37]: Is this needed? Why?

Comment (evenstensberg) [reply package.json:37]: I guess the change from the new version runs install when husky is invoked?

Comment (snitin315) [reply package.json:37]: This is mentioed in the [migration guide](https://github.com/typicode/husky/releases/tag/v9.0.1), `install` is no longer required & deprecated.

Screenshot 2024-07-28 at 9 16 06 PM

**Review by evenstensberg — APPROVED** [review]
lgtm

**Pull request merged**